package.json

```diff
@@ -9,7 +9,7 @@
     "lib": "./lib"
   },
   "scripts": {
-    "_mocha": "mocha --check-leaks --recursive --globals _scratch,sanitizedData",
+    "_mocha": "nyc mocha --check-leaks --recursive --globals _scratch,sanitizedData",
```

Comment:
Does adding `nyc` increase test time by any noticeable (couple seconds or more) amount? I often use `npm run _mocha` to skip the linting done by `npm test`.

Reply:
Without `nyc`:
real    0m42.474s
user    0m24.876s
sys     0m4.684s

With `nyc`:
real    0m49.300s
user    0m27.359s
sys     0m5.182s

Reply:
I'll add a `npm run nyc_mocha` for devops.